storm-dist/binary/src/main/assembly/binary.xml

```diff
@@ -30,38 +30,6 @@
             <useProjectArtifact>false</useProjectArtifact>
             <outputDirectory>lib</outputDirectory>
             <unpack>false</unpack>
-            <excludes>
```

Comment:
Yes, the issue was partly that we didn't shade the libraries properly, but also we had excludes in there that were not needed.  The build would print out lots of warning that we were excluding things that were not a part of the dependency list.